### ISSUE TITLE
Build tess-two for ARM64

### ISSUE THREAD

**Issue by longnguyencmg** [rmtheis/tess-two#121]
I got this error. Can you please help to check?

java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.halosolutions..../base.apk"],nativeLibraryDirectories=[/data/app/com.halosolutions..../lib/arm64, /vendor/lib64, /system/lib64]]] couldn't find "liblept.so"

**Comment by rmtheis** [comment]
Do you have code or steps to reproduce this?

**Comment by longnguyencmg** [comment]
Thank you for your response! Actually this error didn't happen when I tested on my device, I got feedbacks from users who use Galaxy S6.

This is the full stack trace:
Stack trace

java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.....free-1/base.apk"],nativeLibraryDirectories=[/data/app/com.....free-1/lib/arm64, /vendor/lib64, /system/lib64]]] couldn't find "liblept.so" at java.lang.Runtime.loadLibrary(Runtime.java:367) at java.lang.System.loadLibrary(System.java:988) at com.googlecode.tesseract.android.TessBaseAPI.(TessBaseAPI.java:44) at com.......thirdparty.CaptureActivity.initOcrEngine(CaptureActivity.java:667) at com.......thirdparty.CaptureActivity.onResume(CaptureActivity.java:353) at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1271) at android.app.Activity.performResume(Activity.java:6671) at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4030) at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4072) at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3268) at android.app.ActivityThread.access$1000(ActivityThread.java:197) at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1681) at android.os.Handler.dispatchMessage(Handler.java:102) at android.os.Looper.loop(Looper.java:145) at android.app.ActivityThread.main(ActivityThread.java:6891) at java.lang.reflect.Method.invoke(Native Method) at java.lang.reflect.Method.invoke(Method.java:372) at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404) at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)

And when I check in the libs of tess-two after building NDK, there are 3 folders:

1. armeabi-v7a
2. mips
3. x86

I'm sure if you updated the code so code can be complied in arm64.
